index.html

```diff
@@ -3993,15 +3981,190 @@ <h2><span class="chunk">mASt</span> mastering metadata</h2>
+          </aside>
+
+          <aside class="example">
+            Example <span class="chunk">mASt</span> chunk mastering display maximum 
+            luminance using traditional SDR shading techniques at 100 cd/m<sup>2</sup> 
+            (described in ITU-R BT.2035):
+            <table id="mASt-chunk-max-luminance-example" class="numbered simple">
+              <tr>
+                <th>Actual value</th>
+                <th>Stored value</th>
+              </tr>
+
+              <tr>
+                <td>100 cd/m<sup>2</sup></td>
+                <td>100</td>
+              </tr>
+            </table>
+          </aside>
+
+          <aside class="example">
+            Example <span class="chunk">mASt</span> chunk mastering display maximum 
+            luminance using "single-master" SDR shading techniques at 200 cd/m<sup>2</sup> 
+            (described in ITU-R BT.2408 Annex Y):
+            <table id="mASt-chunk-max-luminance-example" class="numbered simple">
+              <tr>
+                <th>Actual value</th>
+                <th>Stored value</th>
+              </tr>
+
+              <tr>
+                <td>200 cd/m<sup>2</sup></td>
+                <td>200</td>
+              </tr>
+            </table>
+          </aside>
+
+
+
+
+        </section>
+      </section>
+      
+
+
+      <section id="mLUM-chunk">
+        <h2><span class="chunk">mLum</span> mastering metadata</h2>
+        <!-- <p>The four decimal values below correspond to the four-byte mLum chunk type field:</p> -->
+
+        <p>The four-byte chunk type field contains the hexadecimal values</p>
+
+        <pre>
+<!-- 109 76 85 109 -->6D 4C 55 6D
+        </pre>
+        
+        <p>If present, the <span class="chunk">mLum</span> chunk identifies 
+           the brightest single pixel or "Maximum Content Light Level" (MaxCLL) for the entire 
+           length of content (every frame) and is typically clipped based on the mastering displays 
+           peak luminance specification. The "Maximum Frame Average Light Level" (MaxFALL) value 
+           is determined by measuring each frames average light level. The frame 
+           with the highest avarage light level becomes the MaxFALL value.</p>
+
+  <p>The following specifies the syntax of the <span class="chunk">mLum</span> chunk:</p>
+
+  <table id="mLum-chunk-syntax" class="numbered simple">
+    <caption>
+      mLum chunk components
+    </caption>
+
+    <tr>
+      <th>Name</th>
+      <th>Size</th>
+      <th>Divisor value</th>
+    </tr>
+
+    <tr>
+      <td>Maximum Content Light Level (MaxCLL)</td>
+      <td>4 bytes</td>
+      <td>0.0001 cd/m<sup>2</sup></td>
+    </tr>
+
+    <tr>
+      <td>Maximum Frame-Average Light Level (MaxFALL)</td>
+      <td>4 bytes</td>
+      <td>0.0001 cd/m<sup>2</sup></td>
+    </tr>
+  </table>
+
+
+  <aside class="note">
+    Below are examples of mLum use to identify MaxCLL and MaxFALL values.</sup>
+  </aside>
+
+<aside class="example">
```

Comment:
This isn't critical, but this line should be indented by 2 spaces.

Reply:
Done